lib/internal/fs/read/context.js

```diff
@@ -15,7 +15,10 @@ const {
 
 const { Buffer } = require('buffer');
 
-const { FSReqCallback, close, read } = internalBinding('fs');
+// It is important to not destructure the binding object, as it will
+// cause the creation context to be undefined, and cause fast API calls
+// to fail.
```

Comment:
This is a fairly sizeable footgun... not in this PR but I wonder if some kind of linting would help us enforce this

Reply:
Agreed. That requires a lot of change. Probably someone with more knowledge on eslint should follow up on that

Reply:
Followed up on https://github.com/nodejs/node/pull/53947